sources/examples/README.rst

```diff
@@ -28,17 +28,17 @@ Executable files, if any, will appear in ``_build/bin/``.
 Some projects have multiple ``.lid``/``.hdp`` files. Feel free to point the compiler at each of them.
 Think of a LID/HDP file as a manifest.
  
-The first time that the project compiles, things will take a long time while the project's dependencies 
+The first time that the project compiles, things will take a long time while the project's dependencies
 are compiled first. Repeated builds will be _much_ faster.
 
 
 Where to start
 --------------
 
-Experiment with the examples in the ``console`` directory. Each of them generates one or more CLI utilities 
+Experiment with the examples in the ``console`` directory. Each of them generates one or more CLI utilities
 that explore various features of the Dylan language.
 
 - ``console/hello-world`` demonstrates defining methods, printing to the console and not needing a "``main()`` function" for Dylan programs.
-- ``console/factorial`` demonstrates integer operations and composing libraries. ``factorial-big.lid`` and `factorial-small.lib`` use the same base code to produce multiple applications with different integer internals.
+- ``console/factorial`` demonstrates integer operations and composing libraries. ``factorial-big.lid`` and `factorial-small.lid`` use the same base code to produce multiple applications with different integer internals.
```

Comment:
Sorry, but one last thing...  There should be two back ticks before factorial-small.lid, rather than one.